arkouda/dataframe.py

```diff
@@ -1397,12 +1395,16 @@ def to_pandas(self, datalimit=maxTransferBytes, retain_index=False):
             warn(msg, UserWarning)
             return None
 
-        # Proceed with conversion if possible, ignore index column
+        # Proceed with conversion if possible
         pandas_data = {}
         for key in self._columns:
             val = self[key]
             try:
-                pandas_data[key] = val.to_ndarray()
+                # in order for proper pandas functionality, SegArrays must be seen as 1d
+                # and therefore need to be converted to list
+                pandas_data[key] = (
+                    val.to_ndarray() if not isinstance(val, SegArray) else val.to_ndarray().tolist()
+                )
```

Comment:
just wanted to note that i think this will be simplified to `val.to_list()` once #1596 is merged. So whichever is merged second will need to investigate this a bit